driver-core/src/test/resources/unified-test-format/retryable-writes/bulkWrite-serverErrors.json

```diff
@@ -0,0 +1,205 @@
+{
```

Comment:
I added these two new unified tests for this, in scope of already existing DRIVERS-1385 ticket.